package-lock.json

```diff
@@ -28,11 +28,116 @@
                 "js-tokens": "^4.0.0"
             }
         },
+        "@google-cloud/dialogflow-cx": {
```

Comment:
Do all of these things need to also go into package.json?  

I'm seeing this error when I try to deploy with rc-apps:
```Error: Cannot find module '@rocket.chat/apps-engine/node_modules/@google-cloud/dialogflow-cx'```

Reply:
I have not found a way to remove that error. If I point the app to the correct location of the module, the app will not deploy.